src/presentation/http/router/note.ts

```diff
@@ -1,5 +1,5 @@
 import { FastifyPluginCallback } from 'fastify';
-import NoteService, { AddNoteOptions } from '@domain/service/note.js';
+import NoteService, { AddNoteOptions, GetNoteByIdOptions } from '@domain/service/note.js';
```

Comment:
if you want a type describing request params, it should not be inside a Domain, because it is related only to communication logic. Somewhere in `/presentation/` I guess.

Reply:
For now, i create it in route